*** tmux.conf.old	2020-01-14 23:35:28.535141286 +0100
--- tmux.conf	2020-01-14 23:35:45.791450233 +0100
***************
*** 12,29 ****
  set -g status-left-length 20
  
  # Window colors
! # set -g status-style bg='#6e6e6e',fg='#ffffff'
! # set -g window-status-current-style bg='#42a4b8'
  
  # Pane colors
! # set -g pane-border-style fg='#6e6e6e'
! # set -g pane-active-border-style bg=default,fg='#42a4b8'
  
  # Prefix + :
! # set -g message-style bg='#42a4b8',fg='#ffffff'
  
  # Prefix + w
! # set -g mode-style bg='#42a4b8',fg='#ffffff'
  
  # e.g. start selection in copy-mode with <space>; automatically set to "vi" if $VISUAL or $EDITOR contains "vi"
  setw -g mode-keys vi
--- 12,29 ----
  set -g status-left-length 20
  
  # Window colors
! set -g status-style bg='#6e6e6e',fg='#ffffff'
! set -g window-status-current-style bg='#42a4b8'
  
  # Pane colors
! set -g pane-border-style fg='#6e6e6e'
! set -g pane-active-border-style bg=default,fg='#42a4b8'
  
  # Prefix + :
! set -g message-style bg='#42a4b8',fg='#ffffff'
  
  # Prefix + w
! set -g mode-style bg='#42a4b8',fg='#ffffff'
  
  # e.g. start selection in copy-mode with <space>; automatically set to "vi" if $VISUAL or $EDITOR contains "vi"
  setw -g mode-keys vi
